all.css

```diff
@@ -19,37 +29,443 @@ body {
   left: 52px;
   gap: 0px;
   opacity: 0px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: auto;
 }
-.navigation {
-  width: 401px;
-  height: 40px;
-  gap: var;
-  opacity: 0px;
+
+.navigation-list {
+  list-style-type: none;
+  display: flex;
 }
 
-.button {
-  text-decoration: none;
+.nav-link {
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 24px;
   color: #f5f8fb;
+  text-decoration-line: none;
+}
+
+.button-discord {
```

Comment:
The following buttons have the same styles. Please create and use one class for them, for example 'button-light' or 'button-secondary'
<img width="78" alt="Screenshot 2024-11-26 at 11 07 35 PM" src="https://github.com/user-attachments/assets/67274e98-f0a2-4e03-ab54-6984b6d11fe2">
<img width="102" alt="Screenshot 2024-11-26 at 11 07 42 PM" src="https://github.com/user-attachments/assets/ea63a572-437e-44c3-b86d-c5b4bdeffbbc">
<img width="82" alt="Screenshot 2024-11-26 at 11 08 03 PM" src="https://github.com/user-attachments/assets/8ed8774d-3677-4c6d-8a2e-81fd18a0ad6a">

Reply:
done

Reply:
Please check these buttons
<img width="240" alt="Screenshot 2024-12-07 at 11 10 24 PM" src="https://github.com/user-attachments/assets/8bcbd546-51d6-46ba-a813-04c4d64b035c">

These elements have the same style properties, so you should use the same class for them

Reply:
And the same for following buttons
<img width="140" alt="Screenshot 2024-12-07 at 11 13 38 PM" src="https://github.com/user-attachments/assets/94369cf5-128b-4bd6-9106-6ade0bde199c">
<img width="59" alt="Screenshot 2024-12-07 at 11 13 43 PM" src="https://github.com/user-attachments/assets/d397df55-5423-4a01-b728-8f1c9a178d3e">
<img width="175" alt="Screenshot 2024-12-07 at 11 13 49 PM" src="https://github.com/user-attachments/assets/2c69d7f4-6cc0-4491-aa00-f57d08ed2680">

Reply:
done